src/constants/tokenIds.ts

```diff
@@ -0,0 +1,16 @@
+export enum CoinGeckoIds {
+  INDEX = 'index-cooperative',
+  DPI = 'defipulse-index',
+  CGI = 'coinshares-gold-and-cryptoassets-index-lite',
+  ETH2XFLI = 'eth-2x-flexible-leverage-index',
+  MVI = 'metaverse-index',
+  BTC2XFLI = 'btc-2x-flexible-leverage-index',
+}
+
+export enum TokenSetsIds {
```

Comment:
heads up: Both of these are not going to be needed almost immediately after launch, and will be included in productTokens